src/components/Countdown/_countdown.scss

```diff
@@ -35,4 +35,11 @@
       gap: $spacing-xsm;
```

Comment:
why we would we want to stop using `--compact` for bidsnapshot timers?  Can you check with @davidicus, it seems like the narrower gap might be important right?

Reply:
but the `--compact` also narrowed the letter spacing, shouldn't we do that at least? (i.e. shouldn't we fix the `--compact` classes to no longer change the font color instead of removing the compact variant from `BidSnapshot`)